.github/workflows/tweet.yml

```diff
@@ -0,0 +1,29 @@
+name: Tweet
+
+on:
+  project_card:
+    types: [moved]
+
+permissions:
+  issues: write
+  repository-projects: write
```

Comment:
iirc we could also just give the tweet step these permissions? Just wondering because parse doesn't necessarily need them, right?

Reply:
Right, will adjust.

Reply:
Yeah, that doesn't seem helpful. Thanks for checking!